deploy/docker/start.sh

```diff
@@ -46,6 +43,12 @@ while [ $IDLE_TIME -lt 5 ]; do
     IDLE_TIME=$(expr $IDLE_TIME + 1)
     if curl http://localhost:${MGMT_PORT}/status >/dev/null 2>&1; then
         IDLE_TIME=0
+        # Print the latest erlang.log
```

Comment:
According to http://erlang.org/doc/man/run_erl.html
If we set `RUN_ERL_LOG_GENERATIONS` to 2, there will be always only one file in place.
I.e either .1 or .2
This should simplify the logic a lot

Reply:
I think `RUN_ERL_LOG_GENERATIONS` should be set by the user, we should not specify the value of `RUN_ERL_LOG_GENERATIONS`

Reply:
But I just found that http://erlang.org/doc/man/run_erl.html :

> run_erl will delete the oldest log file to maintain a "hole" in the file sequences.

I think tail does not exist file will cause an error, do you have any good suggestions?

Reply:
we are by a lot of means hiding OTP from users.
overriding erlang.log in a container isn't very much unfriendly to users especially when we print it to console anyway.

Reply:
if we set to 2, given the 'hole' in the sequence, there will only be one file at a time.
we can fork-tail either `.1` or `.2`.

Reply:
I don't understand, erlang.log loops between two files and N files, I don't think there is much difference

Reply:
I mean, if it loops between 2 files, then a `if` `else` check is enough to find the current file to `tail -f`.
Otherwise we'll have to do all the `ps` `grep` `sed` and `awk` magic which is very much error-prone whenever there is a slight format change in the outputs due to for example docker base image or tooling version upgrade.